packages/wasm-dpp/test/unit/document/Document.spec.js

```diff
@@ -1,36 +1,50 @@
-const DataContractFactory = require('@dashevo/dpp/lib/dataContract/DataContractFactory');
-
-const generateRandomIdentifier = require('@dashevo/dpp/lib/test/utils/generateRandomIdentifier');
-
+const JsDataContractFactory = require('@dashevo/dpp/lib/dataContract/DataContractFactory');
+const JsIdentifier = require('@dashevo/dpp/lib/identifier/Identifier');
+const JsDocument = require('@dashevo/dpp/lib/document/Document');
 const DocumentCreateTransition = require(
   '@dashevo/dpp/lib/document/stateTransition/DocumentsBatchTransition/documentTransition/DocumentCreateTransition',
 );
-
-const Identifier = require('@dashevo/dpp/lib/identifier/Identifier');
-
 const protocolVersion = require('@dashevo/dpp/lib/version/protocolVersion');
 const createDPPMock = require('@dashevo/dpp/lib/test/mocks/createDPPMock');
+const generateRandomIdentifier = require('@dashevo/dpp/lib/test/utils/generateRandomIdentifier');
+const lodash = require('lodash');
 
-const Document = require('@dashevo/dpp/lib/document/Document');
+const generateRandomIdentifierAsync = require('../../../lib/test/utils/generateRandomIdentifierAsync');
+const { default: loadWasmDpp } = require('../../../dist');
 
-const hash = require('@dashevo/dpp/lib/util/hash');
-const serializer = require('@dashevo/dpp/lib/util/serializer');
+let DataContractFactory;
+let DataContractValidator;
+let Identifier;
+let Document;
 
 describe('Document', () => {
-  let hashMock;
-  let encodeMock;
   let rawDocument;
   let document;
   let dataContract;
+  let documentJs;
+  let dataContractJs;
+  let rawDocumentJs;
+  let rawDocumentWithBuffers;
+
+  beforeEach(async () => {
+    ({
+      Identifier, Document, DataContractFactory, DataContractValidator,
+    } = await loadWasmDpp());
+  });
 
   beforeEach(function beforeEach() {
     const now = new Date().getTime();
+    const jsId = generateRandomIdentifier();
+    const id = new Identifier(jsId.toBuffer());
 
-    const ownerId = generateRandomIdentifier().toBuffer();
+    const jsOwnerId = new generateRandomIdentifier();
```

Comment:
That should be `await generateRandomIdentifier();`

Reply:
as we discussed, usage of `async function` tiggers the linter. The error doesn't pop up for other files as they use `this` inside of the function. Therefore, it is necessary to add an exception for the linter.